ocs_ci/ocs/resources/pod.py

```diff
@@ -1032,11 +1032,14 @@ def get_pvc_name(pod_obj):
         pod_obj (str): The pod object
 
     Returns:
-        pvc_name (str): The pvc_name on a given pod_obj
+        (str): The pvc name of a given pod_obj,
+                None if there is no pvc attached
+
     """
-    return pod_obj.get().get(
-        'spec'
-    ).get('volumes')[0].get('persistentVolumeClaim').get('claimName')
+    pvc = pod_obj.get().get('spec').get('volumes')[0].get('persistentVolumeClaim')
+    if pvc:
```

Comment:
Is this check needed?
Since you already use `.get()`, I think it's safe to just call it directly - 
`return pod_obj.get().get('spec').get('volumes')[0].get('persistentVolumeClaim').get('claimName')`
If a PVC attached, it'll return it, otherwise - None. Unless I'm missing something?

Reply:
was changed because it threw errors

Reply:
get('persistentVolumeClaim') is a None, when you do  get('claimName') on it, it fails

Reply:
@ebenahar